### PR TITLE
 v1: define names for unknown/default iota values

### PR DESCRIPTION
**Pull request by mgfcmartinez**
v1: define names for unknown/default iota values

Previously we were using "_", and this wasn't a problem because
protoc-gen-go prefaces enums with the enum name to prevent clashes.
However, this is not the case with other languages (specifically Java).
Give actual names for our unknown/default enum values to prevent protoc
errors during compilation.

While at it, get rid of the ISO 4217 currency enum, as the codes change
frequently and the enum is likely to go out of date.

### CODE REVIEW

**Comment by mgfcmartinez** [comment]
definition changes are [here](https://github.com/openbank/openbank/pull/14/commits/a71d7c9c82d03a64bb0969f5f1e7d4e5b120ba06), the generated files are in a different commit for ease of review. will squash and rebase once approved

**Review by nkristianto — APPROVED** [review]
Thanks for add unknown type for the definition

File: v1/types/types.gunk [hunk 1]
```diff
@@ -195,508 +195,31 @@ const (
...
-	ACTIVE
+	// Active is the active status for a card.
+	Active
 )
 
 type CardAccessStatus int
 
 const (
-	_ CardAccessStatus = iota
+	Unkown CardAccessStatus = iota
```
Comment (Kunde21) [inline v1/types/types.gunk]: Add the full type name here.

**Review by Kunde21 — APPROVED** [review]
One minor nit

File: v1/types/types.gunk [hunk 2]
```diff
@@ -195,508 +195,31 @@ const (
 	Transferwise
 )
 
-// Currency following format specified on ISO 4217
```
Comment (nkristianto) [inline v1/types/types.gunk:198]: btw miggy, why const of currency is deleted ?

**Review by Kunde21 — APPROVED** [review]

File: v1/types/types.gunk (hunk 2 above)
Comment (ghazninattarshah) [reply v1/types/types.gunk:198]: @nkristianto the enum *CurrencyISO4217* is used in ob-services are referred from `transfer` package and not from types. 
@mgfcmartinez CMIIW

**Pull request merged**